EDA-report-covid-19.ipynb

```diff
@@ -0,0 +1,1679 @@
+       "  </tbody>\n",
+       "</table>\n",
+       "<p>5 rows × 50 columns</p>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "  iso_code      continent location        date  total_cases  new_cases  \\\n",
+       "0      ABW  North America    Aruba  2020-03-13          2.0        2.0   \n",
+       "1      ABW  North America    Aruba  2020-03-19          NaN        NaN   \n",
+       "2      ABW  North America    Aruba  2020-03-20          4.0        2.0   \n",
+       "3      ABW  North America    Aruba  2020-03-21          NaN        NaN   \n",
+       "4      ABW  North America    Aruba  2020-03-22          NaN        NaN   \n",
+       "\n",
+       "   new_cases_smoothed  total_deaths  new_deaths  new_deaths_smoothed  ...  \\\n",
+       "0                 NaN           NaN         0.0                  NaN  ...   \n",
+       "1               0.286           NaN         NaN                  0.0  ...   \n",
+       "2               0.286           NaN         0.0                  0.0  ...   \n",
+       "3               0.286           NaN         NaN                  0.0  ...   \n",
+       "4               0.286           NaN         NaN                  0.0  ...   \n",
+       "\n",
+       "   gdp_per_capita  extreme_poverty  cardiovasc_death_rate  \\\n",
+       "0       35973.781              NaN                    NaN   \n",
+       "1       35973.781              NaN                    NaN   \n",
+       "2       35973.781              NaN                    NaN   \n",
+       "3       35973.781              NaN                    NaN   \n",
+       "4       35973.781              NaN                    NaN   \n",
+       "\n",
+       "   diabetes_prevalence  female_smokers  male_smokers  handwashing_facilities  \\\n",
+       "0                11.62             NaN           NaN                     NaN   \n",
+       "1                11.62             NaN           NaN                     NaN   \n",
+       "2                11.62             NaN           NaN                     NaN   \n",
+       "3                11.62             NaN           NaN                     NaN   \n",
+       "4                11.62             NaN           NaN                     NaN   \n",
+       "\n",
+       "   hospital_beds_per_thousand  life_expectancy  human_development_index  \n",
+       "0                         NaN            76.29                      NaN  \n",
+       "1                         NaN            76.29                      NaN  \n",
+       "2                         NaN            76.29                      NaN  \n",
+       "3                         NaN            76.29                      NaN  \n",
+       "4                         NaN            76.29                      NaN  \n",
+       "\n",
+       "[5 rows x 50 columns]"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "covid.head(n=5)"
```

Comment:
If we leave just `covid` without `head()` method and without specifying the number of rows we want to see we will see the top and bottom of the dataset, specifically 5 top rows and 5 bottom rows all in one dataframe. 
Then, we wouldn't need `covid.tail(n=5)` below. 

This is just a suggestion for a cleaner code, I'm good with what you have here otherwise.